### DISCUSSION TITLE
Linkquality values in console and Zigbee Map

### DISCUSSION THREAD

**Discussion by LucReynders** [arendst/Tasmota#11518] [Q&A] [answered]
This is the feedback in the console after sending a command to a device (this case a BasicZBR3) :

```lua
      09:02:25.633 CMD: zbsend {"device":"ZB1","send":{"Power":2}}
      09:02:25.642 MQT: stat/ZBBridge/RESULT = {"ZbSend":"Done"}
      09:02:26.390 MQT: tele/ZBBridge/ZB1/SENSOR = {"ZbReceived":{"ZB1":    {"Device":"0x3C0E","Name":"ZB1","Power":1,"Endpoint":1,"LinkQuality":13}}}
      09:02:28.142 CMD: zbsend {"device":"ZB1","send":{"Power":2}}
      09:02:28.150 MQT: stat/ZBBridge/RESULT = {"ZbSend":"Done"}
      09:02:28.907 MQT: tele/ZBBridge/ZB1/SENSOR = {"ZbReceived":{"ZB1":{"Device":"0x3C0E","Name":"ZB1","Power":0,"Endpoint":1,"LinkQuality":21}}}
      09:02:29.757 CMD: zbsend {"device":"ZB1","send":{"Power":2}}
      09:02:29.765 MQT: stat/ZBBridge/RESULT = {"ZbSend":"Done"}
      09:02:30.515 MQT: tele/ZBBridge/ZB1/SENSOR = {"ZbReceived":{"ZB1":{"Device":"0x3C0E","Name":"ZB1","Power":1,"Endpoint":1,"LinkQuality":18}}}
      09:02:32.467 CMD: zbsend {"device":"ZB1","send":{"Power":2}}
      09:02:32.475 MQT: stat/ZBBridge/RESULT = {"ZbSend":"Done"}<!-- Thanks for reporting a problem for this project. READ 
```
The device is close to another powered Zigbee device (not battery) but the link quality seems to be low (also with other devices). In fact this value less than 20 should indicate a bad link and lead to unreliable connections.

How should I read/interprete this link quality ?

Below the Zigbee map :  are the "numbers" on the connection lines the link quality values ?  These values seem to be more correct than the one in the console.

<img width="821" alt="Schermafbeelding 2021-03-30 om 10 15 50" src="https://user-images.githubusercontent.com/45813325/112956891-356a6900-9141-11eb-875f-1755d6aa38b5.png">

Anyway ... Tasmota on the Sonoff Zigbee Bridge  is great.  
I will move devices from Wifi-Tasmota to the Zigbee Bridge.

**Comment by ascillato2** [comment, marked as answer]
Please, address this to the [Tasmota Support Chat](https://discord.gg/Ks2Kzd4). The chat is a better and more dynamic channel for helping you.

Thanks.

---------------------------

### Support Information 
See [Docs](https://tasmota.github.io/docs/) for more information.
See [Chat](https://discord.gg/Ks2Kzd4) for more user experience.
See [Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)
